doc/source/reference/arrays.datetime.rst

```diff
@@ -368,7 +369,7 @@ times in UTC. By default, creating a datetime64 object from a string or
 printing it would convert from or to local time::
 
     # old behavior
-    >>> np.datetime64('2000-01-01T00:00:00')
+    >>> np.datetime64('2000-01-01T00:00:00')  # doctest: +SKIP
```

Comment:
Why the skip?

Reply:
Because this is a historical snippet of what _used_ to happen, not what does happen

Reply:
Yes, it really is a case of "mandatory skip". Should we document the reasons for skipping doctests somewhere? For instance (hoping the comment syntax allows for it)

    >>> ... #  doctest: +SKIP # the output shows the old and deprecated result

or

    >>> ... #  doctest: +SKIP # output of older NumPy versions

or even somewhere else (another file in `doc/`).

Reply:
Sorry, it is fine the way it is. I missed the "old behaviour" comment.